config/routes.rb

```diff
@@ -1,9 +1,9 @@
-# frozen_string_literal: true
-
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
-      resources :reservations, only: [:index, :new, :create] do
+      resources :users, only: [:create] do
+        resources :reservations, only: [:index, :new, :create] do
```

Comment:
The new path in reservations is not actually needed, there's no method by that name in the controller.

Reply:
That's true, I'll delete this on `dev` branch.